README.md

```diff
@@ -1,3 +1,4 @@
-# Project A / Group 1 - Structured Data [![Join the chat at https://gitter.im/MusicConnectionMachine/StructuredData](https://badges.gitter.im/MusicConnectionMachine/StructuredData.svg)](https://gitter.im/MusicConnectionMachine/StructuredData?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+# Project A / Group 1 - Structured Data
 
+[![Join the chat at https://gitter.im/MusicConnectionMachine/StructuredData](https://badges.gitter.im/MusicConnectionMachine/StructuredData.svg)](https://gitter.im/MusicConnectionMachine/StructuredData?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
```

Comment:
Sorry , I do not understand this comment . When I tried visiting the URL it did not open the usual git hub pages. Please explain your comment a bit more.